packages/next-swc/crates/wasm/Cargo.toml

```diff
@@ -23,7 +23,6 @@ path-clean = "0.1"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 tracing = { version = "0.1.37", features = [
-  "release_max_level_off",
   "max_level_off",
```

Comment:
```suggestion
```

This need to be removed too, doesn't it?

Reply:
Oops, thank you!